content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -46,73 +47,61 @@ monitoring statistics by default:
-
-This video shows how to configure and run a Google Cloud Monitoring backed Heapster:
-
-[![how to setup and run a Google Cloud Monitoring backed Heapster](https://img.youtube.com/vi/xSMNR2fcoLs/0.jpg)](https://www.youtube.com/watch?v=xSMNR2fcoLs)
-
-
-{{< figure src="/images/docs/gcm.png" alt="Google Cloud Monitoring dashboard example" title="Google Cloud Monitoring dashboard example" caption="This dashboard shows cluster-wide resource usage." >}}
+* [Elasticsearch and Kibana (Elastic Stack)](https://www.elastic.co/products/elastic-stack) The Elastic stack has a robust feature set for
+ logging, reporting, and visualizing log data.
+
+* [Google Stackdriver](https://cloud.google.com/stackdriver/) is a hosted monitoring service you can use to
+visualize and alert on important metrics in your application. 
+
+* [Prometheus](https://prometheus.io) can natively monitor kubernetes, nodes, and 
+  prometheus itself. The [Prometheus 
+  Operator](https://coreos.com/operators/prometheus/docs/latest/) simplifies 
+  Prometheus setup on Kubernetes, and allows you to serve the custom metrics API 
+  using the [Prometheus 
+  adapter](https://github.com/directxman12/k8s-prometheus-adapter). Prometheus 
+  provides a robust query language and a built-in dashboard for querying and 
+  visualizing your data. Prometheus is also a supported data source for 
+  [Grafana](https://prometheus.io/docs/visualization/grafana/).
+
+* [Sysdig](http://sysdig.com) provides full spectrum container and platform 
+  intelligence, and is a true container native solution. Sysdig pulls together 
+  data from system calls, Kubernetes events, Prometheus metrics, statsD, JMX, and 
+  more into a single pane that gives you a comprehensive picture of your 
+  environment. Sysdig also provides an API to query for providing robust and 
+  customizable solutions. Sysdig is built on Open Source. [Sysdig and Sysdig 
+  Inspect](https://sysdig.com/opensource/inspect/) give you the ability to freely 
+  perform troubleshooting, performance analysis and forensics. 
 
 ## CronJob monitoring
 
-### Kubernetes Job Monitor
-
-With the [Kubernetes Job Monitor](https://github.com/pietervogelaar/kubernetes-job-monitor) dashboard a Cluster Administrator can see which jobs are running and view the status of completed jobs.
-
-### New Relic Kubernetes monitoring integration
-
-[New Relic Kubernetes](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/kubernetes-monitoring-integration) integration provides increased visibility into the performance of your Kubernetes environment. New Relic's Kubernetes integration instruments the container orchestration layer by reporting metrics from Kubernetes objects. The integration gives you insight into your Kubernetes nodes, namespaces, deployments, replica sets, pods, and containers.
+* With the [Kubernetes Job Monitor](https://github.com/pietervogelaar/kubernetes-job-monitor) dashboard, a Cluster Administrator can see which
```

Comment:
Per #15576, this content no longer meets minimum standards. Remove L100-101.

Reply:
removed